### PR TITLE
Riskfactor component

### PR DESCRIPTION
**Pull request by KamilBartczak03**
## Implementation guide

```
<MessageBox>
    <RiskfactorCheckbox v-model="test" :riskFactors="risk_factors"/>
</MessageBox>

const componentValue = ref([])
const risk_factors = reactive([])


fetch("https://api.infermedica.com/v3/suggest", {
  method: "POST",
  headers: {
    "Content-Type": "application/json",
    "App-Id": import.meta.env.VITE_APP_ID,
    "App-Key": import.meta.env.VITE_APP_KEY,
  },
  body: JSON.stringify({
    suggest_method: "demographic_risk_factors",
    age: {
      value: 18,
    },
    sex: "female",
    evidence: []
  }),
}).then(e => e.json()).then(e => e.forEach(el => risk_factors.push(el)))
```

### CODE REVIEW

File: src/components/RiskfactorCheckbox.vue [hunk 1]
```diff
@@ -0,0 +1,102 @@
+<template>
+    <div class="checkbox__container">
```
Comment (patrykkurzeja91) [inline src/components/RiskfactorCheckbox.vue:2]: those indentations look like 4 spaces - don't You use 2?

File: src/components/RiskfactorCheckbox.vue [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+                id: rf.id,
+                source: "suggest",
+                choice_id: computeEnabled(rf.id) ? "present" : "unknown"
+            }
+        })
+        emit("update:modelValue", emitPayload)
+    }
+
+    const rfindex = selected.findIndex(rfid => rfid === id)
+    if (rfindex === -1) {
```
Comment (patrykkurzeja91) [inline src/components/RiskfactorCheckbox.vue:54]: could be a simple ternary

File: src/components/RiskfactorCheckbox.vue (hunk 1 above)
Comment (DanielHajduk98) [reply src/components/RiskfactorCheckbox.vue:2]: Linter should have fixed it.

**Review by patrykkurzeja91 — APPROVED** [review]
LG

**Pull request merged**